openslides_backend/action/actions/meeting/import_.py

```diff
@@ -430,3 +519,91 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
+        """
+        1. Method checks instance for valid migration_index
+        2. convert the instance json-data to datastore create events
+        3. do the migrations
+        4. get the migrated events from migration wrapper
+        5. Convert only create events back to json-data
+        """
+        start_migration_index = instance.get("meeting", {}).pop(
+            "_migration_index", None
+        )
+        if not start_migration_index or start_migration_index < 0:
+            raise ActionException(
+                f"The data must have a valid migration index, but '{start_migration_index}' is not valid!"
+            )
+        backend_migration_index = get_backend_migration_index()
+        if backend_migration_index < start_migration_index:
+            raise ActionException(
+                f"Your data migration index '{start_migration_index}' is higher than the migration index of this backend '{backend_migration_index}'! Please, update your backend!"
+            )
+        if backend_migration_index > start_migration_index:
+            migration_wrapper = MigrationWrapper(
+                verbose=True,
+                memory_only=True,
+            )
+            organization = self.datastore.get(
+                FullQualifiedId(Collection("organization"), ONE_ORGANIZATION),
+                [
+                    "committee_ids",
+                    "active_meeting_ids",
+                    "archived_meeting_ids",
+                    "template_meeting_ids",
+                    "resource_ids",
+                    "organization_tag_ids",
+                ],
+            )
+            committee = self.datastore.get(
+                FullQualifiedId(Collection("committee"), instance["committee_id"]),
+                ["meeting_ids"],
+            )
+            models = {
+                f"organization/{ONE_ORGANIZATION}": organization,
+                f"committee/{instance['committee_id']}": committee,
+            }
+            migration_wrapper.set_additional_data(
+                self.create_import_create_events(instance),
+                models,
+                start_migration_index,
+            )
+            migration_wrapper.execute_command("finalize")
+            migrated_events = migration_wrapper.get_migrated_events()
+            instance = self.create_instance_from_migrated_events(
+                instance, migrated_events
+            )
+        return instance
+
+    def create_instance_from_migrated_events(
+        self, instance: Dict[str, Any], migrated_events: List[BaseEvent]
+    ) -> Dict[str, Any]:
+        data: Dict[str, Dict] = defaultdict(dict)
+        for event in migrated_events:
+            collection, id_ = collection_and_id_from_fqid(event.fqid)
+            if event.type == CreateEvent.type:
+                data[collection].update({str(id_): event.data})
+            elif event.fqid.split("/")[0] in ("organization", "committee", "user"):
```

Comment:
Use `collection_from_fqid` here (or at least the `KEYSEPARATOR` constant)

Reply:
> Use `collection_from_fqid` here (or at least the `KEYSEPARATOR` constant)

done